uelc/templates/quizblock/edit_question.html

```diff
@@ -19,53 +21,26 @@
 {% endblock %}
 
 {% block js %}
-    <script src="//ajax.googleapis.com/ajax/libs/jquery/1.7.2/jquery.min.js" type="text/javascript" ></script>
-    <script src="//ajax.googleapis.com/ajax/libs/jqueryui/1.8.18/jquery-ui.min.js" type="text/javascript"></script>
-
-    <script type="text/javascript">
-        var saveOrder = function() {
-            var url = "{% url 'reorder-answer' question.id %}?";
-            var worktodo = 0;
-            jQuery("#answers li").each(function(index,element) {
-               worktodo = 1;
-               var id = jQuery(element).attr('id').split("-")[1];
-               url += "answer_" + index + "=" + id + ";";
-            });
-            if (worktodo == 1) {
-                var req = new XMLHttpRequest();
-                req.open("POST",url,true);
-                req.setRequestHeader("X-CSRFToken", '{{csrf_token}}');
-                req.send(null);
-            }
-        };
-        
-        jQuery(document).ready(function () {
-            jQuery("#answers").sortable({
-                containment : 'parent'
-                ,axis : 'y'
-                ,tolerance: 'pointer'
-                ,activeClass: 'dragging'
-                ,handle: '.draghandle'
-                ,stop: function (event,ui) { saveOrder();}
```

Comment:
Thanks for cleaning this up. To be honest... I was not really aware that quizblock-admin.js existed